classes/Form/ConfigurationForm.php

```diff
@@ -159,6 +174,11 @@ public function generate()
         $helper->fields_value['GA_CROSSDOMAIN_ENABLED'] = Configuration::get('GA_CROSSDOMAIN_ENABLED');
         $helper->fields_value['GA_ANONYMIZE_ENABLED'] = Configuration::get('GA_ANONYMIZE_ENABLED');
 
+        $gaCancelledStates = json_decode(Configuration::get('GA_CANCELLED_STATES'));
```

Comment:
json_encode returns an StdClass instance, are you sure? Shouldn't be an array?

Reply:
Fixed